assets/server/static/js/popup/main.js

```diff
@@ -12,6 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+document.addEventListener("DOMContentLoaded", function () {
+  var categorySelect = document.getElementById("category");
+  var reasonInput = document.getElementById("reason");
+
+  // Update the reason input's placeholder with the selected category's hint.
+  function updatePlaceholder() {
+    var selectedOption = categorySelect.options[categorySelect.selectedIndex];
+    reasonInput.placeholder = selectedOption.getAttribute("hint");
+  }
+
+  // Call the function when the page loads.
+  updatePlaceholder();
```

Comment:
You should guard against these objects not being found before calling `updatePlaceholder`.

Reply:
updated in the latest version.